src/Tag/Tag.php

```diff
@@ -0,0 +1,48 @@
+<?php
+
+declare(strict_types=1);
+
+namespace Yiisoft\Html\Tag;
+
+use Yiisoft\Html\Html;
+
+abstract class Tag
+{
+    protected array $attributes = [];
+    
+    final protected function __construct()
```

Comment:
Does that work with private?

```suggestion
    final private function __construct()
```

Reply:
Yes, I did it